### PR TITLE
Two small fixes to enable reconnecting and prevent forked process from hanging at 100% cpu

### PR DESCRIPTION
**Pull request by PaulWieland**

### CODE REVIEW

**Comment by Neustradamus** [comment]
@PaulWieland: Applied, thanks!

File: ap2-receiver.py [hunk 1]
```diff
@@ -440,6 +441,12 @@ def do_TEARDOWN(self):
         self.send_header("Server", self.version_string())
         self.send_header("CSeq", self.headers["CSeq"])
         self.end_headers()
+        
+        # Erase the hap() instance, otherwise reconnects fail
+        self.server.hap = None
+
+        # terminate the forked event_proc, otherwise a zombie process consumes 100% cpu
+        self.event_proc.terminate()
```
Comment (glmnet) [inline ap2-receiver.py:449]: This seems to cause the tcp connection to be closed. Hence when skipping a track forward airplay disconnects.

**Review by glmnet — COMMENTED** [review]
This is giving issues with buffered audio

**Comment by Neustradamus** [comment]
@systemcrash: Have you seen the @glmnet comment?

File: ap2-receiver.py (hunk 1 above)
Comment (systemcrash) [reply ap2-receiver.py:449]: Well, hanging processes seem worse than disconnects at fast forward. But this is a good find. Do your instincts have any proposals about how to reconcile the CPU fix and a disconnect fix?

**Pull request merged**